RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,34 @@
+# Release Checklist
+
+This is a list of the things that need to happen during a release.
+
+1. Open the associated milestone. All issues and PRs should be closed. If
+  they are not you should reassign all open issues and PRs to future
+  milestones.
+1. Go through the commit history since last release. Ensure that all PRs
+  that have landed are marked with the milestone.
+1. Go through the closed PRs in the milestone. Each should have a changelog
+  label indicating if the change is docs, fix, feature, or maintenance. If
+  there is a missing label, please add one.
+1. Choose an emoji for the release. Try to make it some sort of transition
+  from the previous releases emoji (point releases can be a little weirder).
+1. Create a new branch "#.#.#" where "#.#.#" is the release's version.
+1. Add this release to the `CHANGELOG.md`. Use the structure of previous 
+  entires.
+1. Update the version in `Cargo.toml`.
+1. Update the version number and date in `docs/index.html`.
+1. Run `cargo update`.
+1. Run `cargo build`.
```

Comment:
I think it would be a good idea to run `cargo test` as well

Reply:
yes! i definitely do that i just forgot to add it!